components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -56,6 +62,10 @@
  */
 public class DataToCellConverter
 {
+    /** Logging helper object. */
+    @Inject
```

Comment:
This is not a component, so `@Inject` won't work. Use this instead:

    private Logger logger = LoggerFactory.getLogger(DataToCellConverter.class);